### ISSUE TITLE
No module named PySide - Maya 2017

### ISSUE THREAD

**Issue by thnewlands** [ZGeng/VertAnimToTex#2]
Error: line 1: ImportError: file ----Documents/maya/2017/scripts\VertAnimToTex\UI.py line 1: No module named PySide

Script needs to be updated to Pyside2!! 

https://fredrikaverpil.github.io/2016/07/25/dealing-with-maya-2017-and-pyside2/

**Comment by ZGeng** [comment]
@thnewlands Thanks a lot for your reminder and the link, I'll look into it and try to update the code. Thanks again.

**Comment by stealthgg** [comment]
Yes please! This is the closest thing we have in Maya to Jonathan Lindquist's VertexAnimationTools made in Maxscript, aside from this somewhat incomplete (and also outdated) tool:
https://forums.unrealengine.com/community/community-content-tools-and-tutorials/85150-vertexanimationtools-for-maya-wip-nearly-completed

**Comment by jojjo223** [comment]
Still not working. Tested on Maya 2019 and giving the same error, _"...Documents/maya/scripts\VertAnimToTex\UI.py line 1: No module named PySide"_

**Comment by toothbrush-z** [comment]
Seems pyside module name got updated on maya python please reference to the web page https://help.autodesk.com/view/MAYAUL/2017/ENU/?guid=__files_GUID_3F96AF53_A47E_4351_A86A_396E7BFD6665_htm

**Comment by sbissonnette** [comment]
There you go, this is working well in Maya2022
[VertAnimToTex.zip](https://github.com/ZGeng/VertAnimToTex/files/6674064/VertAnimToTex.zip)

**Comment by sbissonnette** [comment]
The folder icon was not working... I'm doing the whole loop again and re-upload the fix...

**Comment by sbissonnette** [comment]
Et Voila! @ZGeng 

The full loop in Maya2022 is working (Python3)

Have fun!

[VertAnimToTex.zip](https://github.com/ZGeng/VertAnimToTex/files/6674384/VertAnimToTex.zip)

**Comment by LuckySawa** [comment]
Maya 2022, the generated textures are all black, but the output window says success, don't know why
Anyone can help?

**Comment by yupixleandro** [comment]
im having the black texture too in Maya 2020  =((

**Comment by RoyLivne** [comment]
Same here in maya :-( would we good if can be fixed please.....
